src/app/pages/dash/character-details/character-details.component.ts

```diff
@@ -27,6 +27,11 @@ export class CharacterDetailsComponent implements OnInit {
     storiesList: [],
   };
 
+  public seriesListFilter = this.characterDetailList.seriesList;
```

Comment:
Isso está funcionando como esperado? Essa declaração é feita assim que o componente renderiza. Ao popular this.characterDetailList, os outros arrays seriesListFilter, etc. estão sendo populados?

Reply:
Estão sendo populados sim.